rest_framework-stubs/response.pyi

```diff
@@ -22,6 +23,7 @@ class Response(SimpleTemplateResponse):
         exception: bool = ...,
         content_type: str | None = ...,
     ) -> None: ...
+    def __class_getitem__(cls, *args: Incomplete, **kwargs: Incomplete) -> Incomplete: ...
```

Comment:
Please, provide correct types, `Incomplete` is not a good solution :(

Reply:
Thank you for letting me know the shortcomings! Is it okay to change only the 'response' part for that type, or should I revise the other parts as well?

> Please, provide correct types, `Incomplete` is not a good solution :(